nextjs-blog/components/speeddial.js

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import { useState } from "react";
+import Box from "@mui/material/Box";
+import FullscreenOverlay from "../components/transoverlay";
+import SpeedDial from "@mui/material/SpeedDial";
+import SpeedDialIcon from "@mui/material/SpeedDialIcon";
+import SpeedDialAction from "@mui/material/SpeedDialAction";
+import PlayArrowIcon from "@mui/icons-material/PlayArrow";
+import CopyrightIcon from "@mui/icons-material/Copyright";
+import LocalMoviesIcon from "@mui/icons-material/LocalMovies";
```

Comment:
Same here - we can rename to whatever we'd like. Can be helpful in some cases!
`import { LocalMovies as BananaIcon, Copyright as CopyrightIcon } from "@mui/icons-material/Copyright"`